src/plugins/auto_batch/tests/functional/skip_tests_config.cpp

```diff
@@ -24,6 +24,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*VirtualPlugin.*BehaviorTests.*OVHoldersTest.*)",
         // BATCH plugin doesn't support this case
         R"(.*LoadNetworkCreateDefaultExecGraphResult.*)",
+        // BATCH/TEMPLATE plugin doesn't support this case
+        R"(.*OVInferRequestPerfCountersTest.*CheckOperationInProfilingInfo.*)",
```

Comment:
can we implement / emulate this in template plugin?
CC @ilyachur

Reply:
Yes we can.
Why auto batch mentions template plugin?

Reply:
Looks like only in tests.

Reply:
@riverlijunjie  I think here batch is leverage template to cover more tests, am I right?